src/external-actor-on-team-approved-pr.ts

```diff
@@ -0,0 +1,101 @@
+import * as core from "@actions/core";
+import * as httpm from "@actions/http-client";
```

Comment:
Where is this dependency coming from? Looks like we are missing it in package.json